### ISSUE TITLE
As a developer I want to generate a testing report for the code base at the end of sprint 3

### ISSUE THREAD

**Issue by Alessandro100** [Alessandro100/concordia-campus-guide#84]
Metrics:

You must have 70% minimum code coverage from unit tests (or widget tests, etc.)
You must have automated system tests at this point. You will be asked to run live one system test in our meeting. You will be also asked to show the code of some unit tests.

**Comment by wltrf** [comment]
Note: Test coverage report is now automatically generated whenever "npm test" is ran. It is stored in [RootFolder]/coverage. This task also includes describing in plain english the various tests conducted.
